app/javascript/packs/application.js

```diff
@@ -7,15 +7,4 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
-console.log('You\'re doing great!')
-
-
-import React from 'react'
-import ReactDOM from 'react-dom'
-import PropTypes from 'prop-types'
-import App from '../App'
-
-ReactDOM.render(
-  <App/>,
-  document.getElementById('app')
-);
+console.log('Hello World from Webpacker')
```

Comment:
I don't think you want to delete the work from yesterday here, correct? Please be sure to run the following to fix this and re-push:
```
git checkout master
git pull origin master
git checkout nav-bar
git merge master
```